ngageoint/opensphere
# feat(cesium): support altitude modes via ground primitives

This addresses lines/polygons either drawn by the user or from data sources. In addition, `getCoordinateFromPixel` was improved with Cesium's recommendation for handling the terrain vs. ellipsoid cases.

Tests:
General Features
* Test drawing box/circle/polygon in 2D, 3D (no terrain), and 3D (terrain), and all in multiple projections.
* Test loading lines/polys from a file and changing the altitude mode
* Test the case from #381 

Places
* Verify that adding a new place allows you to choose an Altitude Mode and that the preview feature is correctly updated on each change
* Verify that editing the place populates the Altitude Mode dropdown correctly
* Verify that reloading the application displays the geometry correctly
* Verify that exporting the place results in the appropriate `<altitudeMode>` tag being added to the KML
* Verify that importing that KML results in the correct altitude mode
* Add a line and polygon with altitude to places and repeat the above steps


Feature Requests:
- [x] Can places be assigned an altitude if terrain is on in 3D when created via Layers widget (pasting coordinates or clicking on the map)?
- [x] Can non-point places support terrain like polygons? Currently they are flattened.

Discussions:
- [x] Cesium upgrade and new items like `Scene.highDynamicRange`
- [x] Interpolation granularity
